src/index.js

```diff
@@ -24,10 +24,19 @@ function traverseRenderedChildren(internalInstance, callback, argument) {
   }
 }
 
+function extractPublicInfoStack(internalInstance) {
+  const fileName = internalInstance._debugSource &&
+    internalInstance._debugSource.fileName;
+  return {
+    ref: internalInstance._instance,
```

Comment:
Let's rename this field to `publicInstance`. The instance doesn't really have a ref and it can be a bit confusing to overload this term.